src/Downloader.ts

```diff
@@ -685,7 +696,27 @@ class Downloader {
 
     jsConfigVars = jsConfigVars.replace('nosuchaction', 'view') // to replace the wgAction config that is set to 'nosuchaction' from api but should be 'view'
 
-    return { jsConfigVars, jsDependenciesList, styleDependenciesList }
```

Comment:
Why we have only one `getModuleDependencies()` considering that this is render specific?

Reply:
Downloader doesn’t know anything about the chosen render, so we download all modules (both mobiles and desktop) here. More information in https://github.com/openzim/mwoffliner/issues/1923

Reply:
Yes, because `Downloader` should not be specific and problem only "generic" downloading routines. Now we have here very specific (depending from end-points) code in `Downloader` which is not the right place. But I guess this will be fixed in #1923.